src/binwalk/magic/firmware

```diff
@@ -271,7 +271,7 @@
 #
 # Motorola S-Records, from Gerd Truschinski <gt@freebsd.first.gmd.de>
 # Improved by Martin Sundhaug <martinsundhaug@gmail.com>
-0   regex       (S[0-35-9]([0-9A-F]{4})([0-9A-F]{2})+\n)+       Motorola S-Record{many}; binary data in text format, record type:
```

Comment:
the {many}-tag hides output if more than one block matching, the regexes already cover consecutive blocks